tests/testthat/test-process_repos.R

```diff
@@ -1,5 +1,7 @@
 test_that("process_repos() returns results with repo metadata", {
 
+  verbose_off()
```

Comment:
That sets the env var, right? so it has side effects outside the unit test..
I would recommend using `withr::local_envvar` to set enir variables for each test, and they will be unset when the test finishes, we can also make a wrapper in an utils function like set_test_envir_vars() at the beginning of each test if needed/more convenient.